watermarker/src/jvmTest/kotlin/integrationTest/watermarks/TrendmarkIntegrationTestJvm.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2024 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V.
+ *
+ * This work is licensed under the Fraunhofer License (on the basis of the MIT license)
+ * that can be found in the LICENSE file.
+ */
+package integrationTest.watermarks
+
+import de.fraunhofer.isst.trend.watermarker.watermarks.Trendmark
+import de.fraunhofer.isst.trend.watermarker.watermarks.TrendmarkInterface
+import kotlin.reflect.full.companionObjectInstance
+import kotlin.reflect.full.memberProperties
+import kotlin.test.Test
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class TrendmarkIntegrationTestJvm {
+    @Test
+    fun trendmark_tags_consistentTags() {
```

Comment:
Since this test is hard to understand, it would be great to add a short hint that it checks if the tags are unique.